silencer-plugin/src/main/scala/com/github/ghik/silencer/SilencerPlugin.scala

```diff
@@ -11,26 +12,43 @@ class SilencerPlugin(val global: Global) extends Plugin { plugin =>
   val description = "Scala compiler plugin for warning suppression"
   val components: List[PluginComponent] = List(component)
   private var globalFilters = List.empty[Regex]
+  private var pathFilters = List.empty[Regex]
```

Comment:
These could be builders (i.e. `ListBuffer` vals). `.result()` would be called on them when passing to `SuppressingReporter`